server.js

```diff
@@ -1,18 +1,34 @@
 const DB = require('./server/db.js');
 const server = require('./server/app.js');
 const config = require('./server/config.js');
+const fs = require('fs');
+const http = require('http');
+const https = require('https');
+
+let webServer = http.createServer(server.app);
+
+const NODE_ENV = config.get('node_env');
+if(NODE_ENV === 'local.secure') {
```

Comment:
If we change this line to a file existence check, rather than checking for a specific NODE_ENV type, then that should simplify this change.